docs/src/modules/components/ApiPage.js

```diff
@@ -298,13 +298,13 @@ function ApiDocs(props) {
   if (source === '@mui/x-date-pickers' || source === '@mui/x-date-pickers-pro') {
     packages.forEach((pkg) => {
       // e.g. import DatePicker from '@mui/x-date-pickers/DatePicker';
-      imports.push(`import ${componentName} from '${pkg}/${componentName}';`);
+      imports.push(`import { ${pkg.componentName} } from '${pkg.packageName}/${componentName}';`);
```

Comment:
Small fix, I now use the named import even on deep imports to match how the exports actually behave

```diff
- import DatePicker from '@mui/x-date-pickers/DatePicker';
+ import { DatePicker } from '@mui/x-date-pickers/DatePicker';
```

Reply:
This fixes #6695 🙌